catalog/async/catalog-async-postingestplugin/src/test/groovy/org/codice/ddf/catalog/async/processingplugin/ProcessingPostIngestPluginTest.groovy

```diff
@@ -419,42 +502,44 @@ class ProcessingPostIngestPluginTest extends Specification {
     other tests
     */
 
-    def 'test catalogFramework.getResource Exceptions'() {
+    def 'test catalogFramework in resource supplier.get'() {
         given:
-        final String sourceId
-        def metacard = Mock(Metacard) {
-            getSourceId() >> sourceId
+        String sourceId = "sourceId"
+        Metacard metacard = mockMetacard("metacardId", sourceId)
+
+        ResourceResponse resourceResponse = Mock(ResourceResponse)
+        CatalogFramework cf = Mock(CatalogFramework) {
+            1 * getResource(_ as ResourceRequest, sourceId) >> resourceResponse
```

Comment:
I think that this could be moved to a `then:` block and simplified:
```groovy
    def 'test catalogFramework in resource supplier.get'() {
        given:
        String sourceId = "sourceId"
        Metacard metacard = mockMetacard("metacardId", sourceId)

        CatalogFramework cf = Mock(CatalogFramework)

        processingPostIngestPlugin = new ProcessingPostIngestPlugin(cf, processingFramework)

        Supplier<Resource> supplier = processingPostIngestPlugin.getResourceSupplier(metacard, mockSubject())

        when:
        supplier.get()

        then:
        1 * cf.getResource(_ as ResourceRequest, sourceId)
    }
```

Reply:
Updated test.